mappings/icrc/condition-mapping.json

```diff
@@ -1,8 +1,8 @@
 {
-  "id": "icrc-condition-mapping",
```

Comment:
Don't identifiers of mappings conflict if we have mappings with the same in other projects?